### PR TITLE
Reorganize project

### PR DESCRIPTION
**Pull request by dbuksha**
* Moved Dockerfile and docker-compose to docker folder
* Created README.md for docker folder
* Moved root project up

### CODE REVIEW

**Review by eugulixes — CHANGES_REQUESTED** [review]
The headers look in the following way

image

Let's do something about it.
By the way, `##` will be enough for headers.

File: office/settings.py [hunk 1]
```diff
@@ -85,10 +85,10 @@
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.postgresql_psycopg2',
-        'HOST': os.getenv('POSTGRES_HOST', 'database'),
+        'HOST': os.getenv('POSTGRES_HOST', 'localhost'),
```
Comment (eugulixes) [inline office/settings.py]: If you run the project via the Docker container, the value `database` will be suitable for you.
I wouldn't change the value. It doesn't solve any problem.

File: docker/README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+####Requirements
```
Comment (eugulixes) [inline docker/README.md]: You have to add a description above. For example,

```
Here are the Dockerfile and assets related to it to run the project in a Docker container.
```

File: docker/README.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+* Docker
+* Docker-compose
+
+####Getting Started
+
+1. Rename **.env.example** to **.env**.
```
Comment (eugulixes) [inline docker/README.md]: Use `#` instead of `1`, `2`, etc.

Comment (eugulixes) [inline docker/README.md]: Rename ... into ...

File: docker/README.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+2. Change `ALLOWED_HOSTS=''` to `ALLOWED_HOSTS=127.0.0.1` in **.env**
+3. Run `docker-compose up`
+4. Create super user with `docker exec -it officemanager_web_1 python3 manage.py createsuperuser`
```
Comment (eugulixes) [inline docker/README.md]: superuser

File: docker/README.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+5. Open `http://127.0.0.1/admin/` to be sure that everything works
+
+
+####Deleting project
+If you need to delete the project (for example, to install it again) you need to do the following:
+1. Stop and remove all project containters with `docker-compose down` in project's root folder
```
Comment (eugulixes) [inline docker/README.md]: `directory` instead of `folder`. We've got used to the Unix-like operating systems=)

File: docker/README.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+2. Delete directory **dbdata**
+3. Delete all Docker images, which were installed or composed:  `docker rmi officemanager_web postgres python:3.5 redis`
```
Comment (eugulixes) [inline docker/README.md]: Delete all _the_ Docker images _related to the project by invoking_ ... (without `:`)

File: office/settings.py (hunk 1 above)
Comment (dbuksha) [reply office/settings.py]: It was by mistake

**Review by eugulixes — APPROVED** [review]

**Pull request merged**